ci/docker-rust/Dockerfile

```diff
@@ -36,5 +36,6 @@ RUN set -x \
  && cargo install mdbook-linkcheck \
  && cargo install svgbob_cli \
  && cargo install --git https://github.com/rustwasm/wasm-pack --rev b4e619c8a13a8441b804895348afbfd4fb1a68a3 \
```

Comment:
off-topic: seems that commit is now included in the latest release: https://github.com/rustwasm/wasm-pack/releases/tag/v0.11.0

Reply:
oh yeah! thank you for pointing this out! I forgot to do it 😞